core/src/main/java/org/mskcc/cbio/maf/MafRecord.java

```diff
@@ -88,57 +87,38 @@
     private int normalDepth;
     private float normalVaf;
 
-	// custom annotator columns
```

Comment:
cleaned up columns that are never referenced when constructing db objects for `mutation`, `mutation_event` imports